templates/haproxy.cfg.j2

```diff
@@ -36,6 +36,11 @@ defaults
   errorfile 504 /etc/haproxy/errors/504.http
 {% endif %}
 
+{% for proxy in haproxy_proxies %}
```

Comment:
There's no structure to this section, i.e. there's nothing here that makes this specific to proxies, it could be used for any arbitrary configuration. How about move it to the end of the file, and call it something like `haproxy_additional_config`?

Reply:
Yes, I believe the role has to allow for any configurations. This makes the role usable outside the scope of the currently intended/supported uses.

I would like to add support for the stats entry later on.

I agree for the name and location. I will update this.